### PR TITLE
add "from __future__ import absolute_import" to custom/enikshay

### PR DESCRIPTION
**Pull request by nickpell**
python-modernize custom/enikshay/ -f libmodernize.fixes.fix_import -w

@dimagi/py3

### CODE REVIEW

File: custom/enikshay/nikshay_datamigration/migrations/0010_followup_changes.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 # Generated by Django 1.10.7 on 2017-04-27 18:33
 from __future__ import unicode_literals
 
+from __future__ import absolute_import
```
Comment (proteusvacuum) [inline custom/enikshay/nikshay_datamigration/migrations/0010_followup_changes.py]: nit, but could these be combined with the previous `from __future__` import?

Comment (nickpell) [reply custom/enikshay/nikshay_datamigration/migrations/0010_followup_changes.py]: yeah, but it's much easier to run the script to add them and leave as is.  if every file in the codebase ends up having both I'll get carpal tunnel from combining them all.

Comment (nickpell) [reply custom/enikshay/nikshay_datamigration/migrations/0010_followup_changes.py]: in the long-run they'll get removed anyway

Comment (proteusvacuum) [reply custom/enikshay/nikshay_datamigration/migrations/0010_followup_changes.py]: lol, ok.

**Review by proteusvacuum — APPROVED** [review]

**Pull request merged**